client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -19,6 +31,8 @@ import {
 	isLoaded as arePackagesLoaded,
 	isFetchError as arePackagesErrored,
 } from 'woocommerce/woocommerce-services/state/packages/selectors';
+import { isEnabled as flagIsEnabled } from 'config';
```

Comment:
Ha, you're right! I guess it's there because at some point the `shipping labels` feature was behind a feature flag. Removed in https://github.com/Automattic/wp-calypso/pull/24976/commits/7f3553cd10652f63acf94ae25cc9650353f51e47